tests/fixtures_regress/tests.py

```diff
@@ -506,22 +506,20 @@ def test_loaddata_no_fixture_specified(self):
                 verbosity=0,
             )
 
-    def test_ticket_20820(self):
+    def test_loaddata_m2m(self):
```

Comment:
```suggestion
    def test_loaddata_with_inherited_m2m(self):
```
I'd not refactor the docstrings in this file